types/index.d.ts

```diff
@@ -709,6 +709,8 @@ declare module 'mongoose' {
     model?: string | Model<any>;
     /** optional query options like sort, limit, etc */
     options?: any;
+    /** optional boolean, set to `false` to allow populating paths that aren't in the schema */
+    strictPopulate?: boolean;
```

Comment:
LGTM, Thanks 👍
Can you add a test case in `test\types\populate.test.ts` to prevent someone from accidentally removing this in the future?

Reply:
Sounds good, will do 👍